scripts/runtestsuite/runtestsuite.py

```diff
@@ -452,9 +509,11 @@ def render_test_scene(args, logger, report_writer, project_directory, project_fi
 # --------------------------------------------------------------------------------------------------
 
 def render_test_scenes(script_directory, args):
-    rendered_scene_count = 0
+    rendered_scenes = 0
```

Comment:
It should remain as it was

`rendered_scenes` suggest the variable contains multiple scenes, such as a list.

```suggestion
    rendered_scene_count = 0
```